internal/validators/validate_test.go

```diff
@@ -0,0 +1,162 @@
+package validators
+
+import (
+	"testing"
+
+	"github.com/go-playground/validator/v10"
+	"github.com/stellar/go/keypair"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseValidationError(t *testing.T) {
```

Comment:
I think it's amazing and super important to have these more complex test cases with dives and etc., but in addition to these, wdyt about adding separate sub tests for each validation error?
I think that way it might be easier to remember to add tests for new validations and we can better test each one individually. For instance, the `Should be greater than or equal` case of `gt` is not tested in this one.

Reply:
+1